pkg/netutil/netutil.go

```diff
@@ -157,7 +156,10 @@ func urlsEqual(ctx context.Context, lg *zap.Logger, a []url.URL, b []url.URL) (b
 	sort.Sort(types.URLs(a))
 	sort.Sort(types.URLs(b))
 	for i := range a {
-		if !reflect.DeepEqual(a[i], b[i]) {
+		ipa:= net.ParseIP(a[i].Hostname())
```

Comment:
What if hostname is a DNS name and not an IP ?

Reply:
Oh... I see the 'resolv' above... but then... what would be the source of corruption for the comparison ?